tool_collections/fastx_toolkit/fasta_clipping_histogram/fasta_clipping_histogram.xml

```diff
@@ -1,18 +1,25 @@
-<tool id="cshl_fasta_clipping_histogram" name="Length Distribution" version="1.0.0">
+<tool id="cshl_fasta_clipping_histogram" name="Length Distribution" version="@VERSION@">
```

Comment:
I was planning on moving it to IUC in galaxyproject/tools-iuc#1413, and of course I can merge in your changes there to avoid reinventing effort.